packages/core/src/utils/transformers/getArrayValue.ts

```diff
@@ -0,0 +1,34 @@
+import { Asset, Entry } from 'contentful';
+import { get } from '../get';
+import { EntityStoreBase } from '@/entity';
+import { isArray } from 'lodash-es';
+
+export function getArrayValue(
+  entryOrAsset: Entry | Asset,
+  path: string,
+  entityStore: EntityStoreBase,
+) {
+  if (entryOrAsset.sys.type === 'Asset') {
+    return entryOrAsset;
+  }
+
+  const arrayValue = get(entryOrAsset, path.split('/').slice(2, -1));
```

Comment:
It looks like the type for `arrayValue` is being inferred as `any` here.  It might be nice to add a type argument here so TS can help catch type errors if this code is updated in the future.

```suggestion
  const arrayValue = get<Array<string | UnresolvedLink<'Entry' | 'Asset'>>>(entryOrAsset, path.split('/').slice(2, -1));
```